tests/fsharp/core/load-script/script.bat

```diff
@@ -44,4 +44,6 @@ echo Test 16=================================================
 "%FSC%" ProjectDriver.fsx --nologo
 ProjectDriver.exe
 del ProjectDriver.exe
+echo Test 17=================================================
```

Comment:
@enricosada has a pending PR that deletes these old test .bat files, but I didn't know about it until after I submitted this and I suppose this might cause a little merge conflict